### PR TITLE
docs: add stepan suvorov to GDE resources

### PR DESCRIPTION
**Pull request by stevermeister**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by petebacondarwin — APPROVED** [review]
Looks good in terms of content.

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -823,6 +823,14 @@
     "groups": ["Angular"],
     "lead": "juleskremer"
   },
+  "stevermeister": {
+    "name": "Stepan Suvorov",
+    "picture": "stevermeister.jpg",
+    "twitter": "stevermeister",
+    "website": "https://stepansuvorov.com/blog/",
+    "bio": "Stepan is a CTO at Studytube, organizer of Angular Kharkiv(Ukraine), co-organizer Dutch Angular Group, teacher at javascript.info, creator of Angular Pro screencast (angular.ninja)",
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json:831]: Maybe:

```suggestion
    "bio": "Stepan is a CTO at Studytube, organizer of Angular Kharkiv(Ukraine), co-organizer of Dutch Angular Group, teacher at javascript.info, creator of Angular Pro screencast (angular.ninja)",
```

**Review by gkalpak — COMMENTED** [review]
The image file is empty. Is that intentional?

**Comment by petebacondarwin** [comment]
> The image file is empty. Is that intentional?

The rich diff indicates the image is there

**Comment by gkalpak** [comment]
> The rich diff indicates the image is there

Oh, interesting. The GitHub UI says the file is empty 🤔

**Review by twerske — APPROVED** [review]
lgtm

**Comment by AndrewKushnir** [comment]
@stevermeister thanks for creating the PR.

FYI the changes are now merged and will reach angular.io website within the next hour or so.

**Pull request closed without merging**